Apsitvarkom.UnitTests/Models/Mapping/CleaningEventMappingTests.cs

```diff
@@ -58,5 +58,33 @@ public void CleaningEventToCleaningEventResponse()
             Assert.That(response.PollutedLocationId, Is.EqualTo(businessLogicObject.PollutedLocationId));
         });
     }
+
+    [Test]
+    public void CleaningEventCreateRequestToCleaningEvent()
+    {
+        var businessLogicObject = new CleaningEvent
+        {
+            Id = Guid.NewGuid(),
+            StartTime = new DateTime(2022, 9, 16, 21, 43, 31).ToUniversalTime(),
+            Notes = "Hello world",
+            PollutedLocationId = Guid.NewGuid()
+        };
+
+        var createModel = new CleaningEventCreateRequest
+        {
+            StartTime = new DateTime(2022, 9, 16, 21, 43, 31).ToUniversalTime(),
+            Notes = "Hello world",
+            PollutedLocationId = Guid.NewGuid()
+        };
+
+        var response = _mapper.Map<CleaningEventCreateRequest, CleaningEvent>(createModel, businessLogicObject);
```

Comment:
in all our mapping tests we check the mapping behavior without colliding two objects (for example as done here - using some `default` instance).
let's change this to also use only `_mapper.Map<CleaningEvent>(cleaningEventCreateRequest);`